locations/spiders/barlouie.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+import scrapy
+
+from locations.items import GeojsonPointItem
+
+
+class BarlouieSpider(scrapy.Spider):
+    name = "barlouie"
+    allowed_domains = ["www.barlouie.com"]
+    start_urls = ["https://www.barlouie.com/locations"]
+    item_attributes = {"brand": "Barlouie", "brand_wikidata": "Q16935493"}
+
+    def parse(self, response):
+        for link in response.xpath("//a[contains(@href, 'locations')]/@href").getall():
+            yield scrapy.Request(
+                "https://www.barlouie.com" + link, self.parse_each_website
+            )
+
+    def parse_each_website(self, response):
+        for ldjson in response.xpath(
+            '//script[@type="application/ld+json"]/text()'
+        ).extract():
+            sanitized_data = self.sanitize_json(ldjson)
+            data = json.loads(sanitized_data)
+            if data["@type"] == "Restaurant":
+                yield self.parse_store(response, data)
```

Comment:
Something like:
```suggestion
                item = LinkedDataParser.parse_ld(data)

                item["ref"] = item["website"] = response.url
                item["image"] = item.get("image", "").replace(
                    "https://www.barlouie.comhttps://images.prismic.io",
                    "https://images.prismic.io",
                )

                yield item
```
Also we could get rid of the Restaurant check by including it in the xpath. Something like `//script[@type="application/ld+json" and contains(text(), 'Restaurant')]/text()`.

Reply:
thank you!! 
Using this made the code shorter and cleaner.